sdk/cognitiveservices/Vision.ComputerVision/CHANGELOG.md

```diff
@@ -1,11 +1,29 @@
 # Release History
 
-## 6.1.0-beta.1 (Unreleased)
+## 7.0.0 (2021-04-12)
```

Comment:
nit:  You may want to consider going with `Unreleased` for the date here to avoid unintentionally signaling a commitment to release.   We typically recommend not assigning a date here until just before doing the release.

Reply:
Thanks. Applied.